migtests/tests/import-file/run-import-file-test

```diff
@@ -94,6 +94,37 @@ main() {
 	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter ',' \
 			--file-table-map "t1_escape_backslash.csv:t1_escape_backslash" --file-opts "escape_char=\\"
 
+	# Test for csv file with backspace as quote character and escape character in text field
+	step "Import data file: test_backspace_char.csv -> test_backspace_char"
+	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter '|' \
+			--file-table-map "test_backspace_char.csv:test_backspace_char" --file-opts "escape_char=\b,quote_char=\b"
+
+	# Test for csv file with backspace as quote character and single quote as escape character
+	step "Import data file: test_backspace_quote_single_quote_escape.csv -> test_backspace_quote_single_quote_escape"
+	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter '|' \
+			--file-table-map "test_backspace_quote_single_quote_escape.csv:test_backspace_quote_single_quote_escape" --file-opts "escape_char=',quote_char=\b"
+
+	# Test for csv file with backspace as quote character and double quote as escape character
+	step "Import data file: test_backspace_quote_double_quote_escape.csv -> test_backspace_quote_double_quote_escape"
+	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter '|' \
+			--file-table-map "test_backspace_quote_double_quote_escape.csv:test_backspace_quote_double_quote_escape" --file-opts "escape_char=\",quote_char=\b"
+
+	# Test for csv file with backspace as escape character and double quote as quote character
+	step "Import data file: test_backspace_escape_double_quote.csv -> test_backspace_escape_double_quote"
+	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter '|' \
+			--file-table-map "test_backspace_escape_double_quote.csv:test_backspace_escape_double_quote" --file-opts "escape_char=\b,quote_char=\""
+
+	# Test for csv file with delimiter as backspace char
```

Comment:
can we have 1 or 2 similar tests of backspace as quote character with `text` format

Reply:
do we use Quote char with `text` format?